src/app/main.ml

```diff
@@ -150,6 +151,7 @@ let begin_program
         Usbmux.Logging.(
           let relay_with =
             R.make_tunnels
+              ~bind_host:bind_host
```

Comment:
in OCaml, if the identifier and parameter name are the same, then you could just do `~bind_host`